### DISCUSSION TITLE
Alpha channel has reduced black levels after copied from grayscale image

### DISCUSSION THREAD

**Discussion by Zo-Bro** [ImageMagick/ImageMagick#3147] [Help]
I have a single-channel image that I am copying into the alpha channel of an rgb image that has no content in the alpha channel. However, when I do that, the resulting alpha channel is "washed out". It does not have the same level of dark gray and black as the original image. How do I fix that? I am using *.tif images. 

Version: ImageMagick-7.0.1-5

The Command Used:
convert rgb_image.tif grayscale_image.tif -alpha off -compose CopyOpacity -composite result.tif

Original Grayscale Image
![image](https://user-images.githubusercontent.com/43020882/105552566-f3442780-5cb8-11eb-9730-5b89faab60ba.png)

The resulting alpha channel 
![image](https://user-images.githubusercontent.com/43020882/105552581-fb9c6280-5cb8-11eb-9201-3e91ef845f89.png)

**Comment by snibgo** [comment]
You say you are using TIF images, but your links are to PNG images. Please link to your actual files of your two inputs.
